zboxcore/sdk/writemarker_mutex.go

```diff
@@ -231,7 +257,11 @@ func (m *WriteMarkerMutex) Unlock(ctx context.Context, sessionID string) error {
 
 	r := resty.New(transport, func(req *http.Request, resp *http.Response, respBody []byte, cf context.CancelFunc, err error) error {
```

Comment:
we should set the default values in resty.New(), and use these options to change the settings. 

It could be implemented simply by setting the default values when create the resty instance, then iterate and apply the options.

Reply:
moved transport to `WithTransport` option. but handle is not optional. so leave it as it is.